.gitignore

```
@@ -39,4 +39,5 @@ glean_app*
 samples/ios/app/.venv/
 .mypy_cache/
 .coverage
-htmlcov/
\ No newline at end of file
+htmlcov/
+raw_sample_xcodetest.log
```

We produce multiple different of these log files.
(raw_sample_xcodetest.log, raw_xcodetest.log, raw_xcodebuild.log, raw_sample_xcodebuild.log)
Let's ignore them all.

```suggestion
*.log
```